prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -559,9 +563,8 @@ workflow_has_parameter GPU_DISPLAY && [[ $NUMAID == 0 ]] && add_W o2-gpu-display
 # AOD
 : ${AOD_INPUT:=$TRACK_SOURCES}
 : ${AODPROD_OPT:=}
-( ! has_detector_matching SECVTX || ! has_detectors_reco ITS || [[ $BEAMTYPE == "cosmic" ]]) && AODPROD_OPT+=" --disable-secondary-vertices"
-( ! has_detector_matching STRK || ! has_detector_matching SECVTX || ! has_detectors_reco ITS || [[ $BEAMTYPE == "cosmic" ]]) && AODPROD_OPT+=" --disable-strangeness-tracking"
-
+[[ ${SECTVTX_ON:-} != "1" ]] && AODPROD_OPT+=" --disable-secondary-vertices "
+AODPROD_OPT+=$STRTRACKING
```

Comment:
Here one cannot use $STRTRACKING since it is defined as "--disable-strangeness-tracker", while for AOD is should be "--disable-strangeness-tracking". It would be good to uniformize AOD and sec vertexing, I think.

Reply:
Hi @shahor02 , 
Am I wrong, or you still try to pass to aod-producer `--disable-strangeness-tracker` in addition to `--disable-strangeness-tracking`?